src/types/ui-customisation-options.ts

```diff
@@ -40,6 +40,7 @@ export type UICustomizationOptions = {
 
   // Applied to both Primary, Secondary Buttons
   borderRadiusButton?: string
+  textTransformButton?: string
```

Comment:
Possible to make it an enum, with the values from here: https://developer.mozilla.org/de/docs/Web/CSS/text-transform

Reply:
@it-ony yep, great suggestion.
replace type